src/app/MessageDef/CommandDataIB.h

```diff
@@ -66,6 +67,17 @@ class Parser : public StructParser
      *          #CHIP_END_OF_TLV if there is no such element
      */
     CHIP_ERROR GetFields(TLV::TLVReader * const apReader) const;
+
+    /**
+     *  @brief Get the provided command reference associated with the CommandData
+     *
+     *  @param [in] apRef    A pointer to apRef
+     *
+     *  @return #CHIP_NO_ERROR on success
+     *          #CHIP_ERROR_WRONG_TLV_TYPE if there is such element but it's not any of the defined unsigned integer types
```

Comment:
This does what other methods do, however I find this inconvenient as we have a ternary return. It would have been great to return the value as "Optional" and then either you get an error or you do not.